cli/src/test.rs

```diff
@@ -388,7 +389,9 @@ fn run_tests(
                     }
                 } else {
                     let mut actual = tree.root_node().to_sexp();
-                    if !has_fields {
+                    let show_fields = opts.show_fields || has_fields;
+
+                    if !show_fields {
```

Comment:
```suggestion
                    if !(opts.show_fields || has_fields) {
```

Reply:
> 
```suggestion
                    let show_fields = opts.show_fields || has_fields;

                    if !show_fields _{_better thinks divide fields#
```
Examp 🥇 former framer